src/plugin/message/Manager/MessageManager.php

```diff
@@ -178,22 +178,25 @@ public function send(Message $message, $setAsSent = true, $sendMail = true)
         return $message;
     }
 
-    public function sendMessageToAbstractRoleSubject(
-        AbstractRoleSubject $subject,
+    public function sendMessage(
         $content,
         $object,
+        ?AbstractRoleSubject $subject = null,
+        $users = [],
```

Comment:
You should only keep an array of AbstractRoleSubject and name it `receivers`.

Reply:
Done :) 
thx